source/use-cases/local-master-key.txt

```diff
@@ -0,0 +1,89 @@
+A. Create a Local Master Key
```

Comment:
If this is intended to be the complete page, get rid of the "A" header, since it's the only "step".

Reply:
This will be part of a Use Case guide. Nick created the document template here: https://github.com/mongodb/docs-ecosystem/commit/4f47fece63aeab1145d7ae5bb8ea6e73ea32d2b6

I suppose I could branch from Nick's branch but am tentative in doing so, given the above template hasn't gone through the PR review process yet. What would you suggest?